experimental/sso-with-skills/javascript_nodejs/README.md

```diff
@@ -0,0 +1,88 @@
+- Update the `SkillAppId` variable in [rootBot/.env](rootBot/.env) with the `AppId` for the skill you created in the previous step
+- (Optionally) Add the `rootBot` `MicrosoftAppId` to the `AllowedCallers` comma separated list in [skillBot/.env](skillBot/.env)
+
+- Setup the 2 AAD applications for SSO as per steps given in [SkillBot AAD](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=srb%2Ccsharp#create-the-azure-ad-identity-application-1) and [RootBot AAD](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=sb%2Ccsharp#create-the-azure-ad-identity-application). You will end up with 2 aad applications - one for the skill consumer and one for the skill.
+- Create an aadv2 connection in the bot registration for the `SkillBot` and fill in values from the aadv2 application created for SSO, as per the [docs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=srb%2Ccsharp#create-azure-ad-connection-1). Update [SkillBot/appsettings.json](SkillBot/appsettings.json) with the `ConnectionName`  
+- Create an aadv2 connection in the bot registration for the `RootBot` and fill in values from the aadv2 application created for SSO, as per the [docs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication-sso?view=azure-bot-service-4.0&tabs=sb%2Ccsharp#create-azure-ad-connection). Update [RootBot/appsettings.json](SkillBot/appsettings.json) with the `ConnectionName`  
+
+- In a terminal, navigate to `samples\javascript_nodejs\82.sso-with-skills\skillBot`
+
+    ```bash
+    cd samples\javascript_nodejs\82.sso-with-skills\skillBot
+    ```
+
+- Install npm modules and start the bot
+
+    ```bash
+    npm install
+    npm start
+    ```
+
+- Open a **second** terminal window and navigate to `samples\javascript_nodejs\82.sso-with-skills\rootBot`
+
+    ```bash
+    cd samples\javascript_nodejs\82.sso-with-skills\rootBot
+    ```
+
+- Install npm modules and start the bot
+
+    ```bash
+    npm install
+    npm start
+    ```
+
+## Testing the bot using Bot Framework Emulator
+
+[Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework Emulator version 4.9.0 or greater from [here](https://github.com/Microsoft/BotFramework-Emulator/releases)
+
+### Connect to the bot using Bot Framework Emulator
+
+- Launch Bot Framework Emulator
+- File -> Open Bot
+- Enter a Bot URL of `http://localhost:3978/api/messages`, the `MicrosoftAppId` and `MicrosoftAppPassword` for the `rootBot`
+- Click Connect
+- Follow the prompts to initiate the token exchange between the `SkillBot` and `RootBot`, resulting in a valid token displayed
```

Comment:
```suggestion
### Connect to the bot using the Bot Framework Emulator

- Launch the Bot Framework Emulator.
- Select **File** > **Open Bot**.
- Enter a Bot URL of `http://localhost:3978/api/messages`, and fill in the **Microsoft App ID** and **Microsoft App password** for the root bot.
- Select **Connect**.
- Follow the prompts to initiate the token exchange between the skill bot and root bot. The bot will display a valid token.
```
The MS writing style for docs for UI interactions is to use bold for UI elements that are labeled (or have hover text), ">" to chain sub-menus, and to use "select" (for _clicking_ a button or choosing a menu item). Again, I'll leave it up to you if you want to follow the same guidelines as for the doc team.